src/dynarec/rv64/dynarec_rv64_helper.c

```diff
@@ -2694,11 +2694,12 @@ void vector_loadmask(dynarec_rv64_t* dyn, int ninst, int vreg, uint64_t imm, int
                     ADDI(s1, xZR, 1);
                     VMV_S_X(vreg, s1);
                     return;
-                case 2:
+                case 2:{
```

Comment:
For the sake of consistent style, you should write as `case 2: {`.
See the file `src/libtools/my_x11_xevent.c` for example.